package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "lambaa",
-    "version": "2.0.0",
+    "version": "2.1.0",
```

Comment:
is this a breaking change? Maybe we need to increment major version?

Reply:
The `NaN` issue might make this a breaking change if people are using `number` for their function parameters already. I can increment the major version to be safe.